platforms/Windows/shared/shared.wxs

```diff
@@ -103,6 +130,10 @@
     <Property Id="TOOLCHAINSVERSIONED">
       <RegistrySearch Root="HKCU" Key="SOFTWARE\!(loc.ManufacturerName)\Installer\$(ProductVersion)\!(bind.Property.UpgradeCode)" Name="ToolchainsVersioned" Type="directory" />
```

Comment:
We should name this to include the variant.

Reply:
that would break the upgrade scenario.